### PR TITLE
[CVE-2021-42550] Update Logback to version 1.2.9

### PR DESCRIPTION
**Pull request by sotty**
In response to:
https://cve.report/CVE-2021-42550

### CODE REVIEW

**Comment by ranma2913** [comment]
Another link: https://nvd.nist.gov/vuln/detail/CVE-2021-42550

File: pom.xml [hunk 1]
```diff
@@ -22,6 +22,9 @@
         <!-- CVE-2021-44228: https://nvd.nist.gov/vuln/detail/CVE-2021-44228 -->
         <!-- CVE-2021-45046: https://nvd.nist.gov/vuln/detail/CVE-2021-45046 -->
         <log4j2.version>2.17.0</log4j2.version>
+        
+        <!-- https://jira.qos.ch/browse/LOGBACK-1591 -->
```
Comment (ranma2913) [inline pom.xml:26]:
```suggestion
        <!-- CVE-2021-42550: https://nvd.nist.gov/vuln/detail/CVE-2021-42550 -->
```

File: pom.xml [hunk 2]
```diff
@@ -192,19 +195,19 @@
         <dependency>
             <groupId>ch.qos.logback</groupId>
             <artifactId>logback-classic</artifactId>
-            <version>1.2.3</version>
+            <version>${logback.version}</version>
```
Comment (ranma2913) [inline pom.xml:198]: Thanks. I really like using a property here since there's several dependencies with the same version.

**Review by ranma2913 — CHANGES_REQUESTED** [review]

File: pom.xml (hunk 1 above)
Comment (sotty) [reply pom.xml]: thank you - I have updated the link

Comment (ranma2913) [reply pom.xml]: Great. Thanks

**Review by ranma2913 — APPROVED** [review]

**Comment by sotty** [comment]
@ranma2913 @fantavlik Are there any next steps, or concerns, on getting this PR approved and merged? 
Thank you in advance

**Comment by sotty** [comment]
I have found and signed the contributor agreement - should I also open an Issue for this PR?

**Comment by sotty** [comment]
Rebased + squashed to keep the history linear

**Comment by fantavlik** [comment]
Thanks @sotty! This is out as v1.11.4 now.

**Pull request merged**